exercises/list-ops/example.rb

```diff
@@ -0,0 +1,67 @@
+class ListOps
```

Comment:
Hi @rhoprhh! Since you are adding these methods in your own class it is ok to have methods named `map` and `concat` inside it. Any motive to use your own names?